input/templates/pop.hbs

```diff
@@ -58,6 +58,9 @@
               <div class="jumbotron-row">
                 {{> catalog/jumbotron jumbotron=content.jumbotron}}
               </div>
+              {{!-- <div class="search-results-row">
+                {{> catalog/search-result search-result=content.searchResult}}
+              </div> --}}
               <div class="col-xs-4 hidden-xs text-left">
```

Comment:
I wouldn't see it as a feature, because it is enabled based on the data we are providing. When search is enabled it doesn't mean we want to display search results (that doesn't mean we won't want a feature search in the future).
@JulianSamarjiev then what you have to do is checking if we have `searchResults`, in which case we show this, otherwise we show the jumbotron.
It's basically a matter of semantics...